openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -25,6 +25,12 @@ class CourseOverview(django.db.models.Model):
     a course as part of a user dashboard or enrollment API.
     """
 
+    # IMPORTANT: Bump this whenever you modify this model and/or add a migration.
+    VERSION = 1
```

Comment:
One other concern, if the new servers clear the table before they're in the load balancer, the cache will just get repopulated by the old servers, defeating the purpose.  Our solution needs to ensure that the new servers are the ones repopulating the cache.  SO, version numbers (or git inspection) would solve that.  Doing it on server boot would not.